Automattic/wp-calypso
# InlineHelp: move selectors into `./inline-help/selectors` folder

#### Changes proposed in this Pull Request

This PR follows https://github.com/Automattic/wp-calypso/pull/43404 in order to improve the modularization of the global state.

#### Testing instructions (almost same as #43404)

* Execute Unit tests for all global selectors.

```
> cd test/client
> yarn run test-client client/state/inline-help/test/selectors.js
```

```
$ check-npm-client && jest -c=test/client/jest.config.js client/state/inline-help/test/selectors.js
 PASS  client/state/inline-help/test/selectors.js

Test Suites: 1 passed, 1 total
Tests:       6 passed, 6 total
Snapshots:   0 total
Time:        3.595 s, estimated 4 s
Ran all test suites matching /client\/state\/inline-help\/test\/selectors.js/i.
✨  Done in 11.04s.
```
* Check that either `Get Help` section in the customer home page is working as expected, as well as the `Inline Help Popover`. Check it records tracks events too.

* A visual code inspection is also a welcomed to test this PR
